akka-actor-typed-tests/src/test/scala/akka/actor/typed/AskSpec.scala

```diff
@@ -44,7 +44,8 @@ class AskSpec extends ScalaTestWithActorTestKit("""
       foo.replyTo ! "foo"
       Behaviors.same
     case (_, Stop(r)) ⇒
-      r ! ()
+      val x = ()
+      r ! x
```

Comment:
Helps us not having to disable for all akka-actor-typed-tests just for this one point.

Reply:
Same here: `(())`